### PR TITLE
Schema diagram

### PR DESCRIPTION
**Pull request by momotsow**
### Created  database schema from the given Clinic diagram

- Created a file named schema_based_on_diagram.sql where we implemented the database from the diagram.
- Joined tables from many-to-many relationships.
- Added the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        med_history_id INTEGER NOT NULL,
+        treatment_id INTEGER NOT NULL,
+        CONSTRAINT PK_Treatment_histories PRIMARY KEY (med_history_id, treatment_id)
+    );
+
+ALTER TABLE treatment_histories
+ADD FOREIGN KEY (med_history_id) REFERENCES medical_histories (id);
+
+ALTER TABLE treatment_histories
+ADD FOREIGN KEY (treatment_id) REFERENCES treatments (id);
```
Comment (sja-thedude) [inline schema_based_on_diagram.sql]: - [ ] Kindly add foreign keys for all tables as per the requirements by using references table_name(id) in column definition.

- [ ] Kindly make sure to have indexes created for all foreign keys as per the requirements.

- [ ] Kindly make sure that all "join table" for the many-to-many relastionships is created.
Example: create a table between `medical_histories` and `treatments`

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @momotsow & @lmunozm1702, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Correct gitflow
✔️ Profesional Readme

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (momotsow) [reply schema_based_on_diagram.sql]: Hi @sja-thedude , Thank you for your review.

We have already created the  "join table" for the many-to-many relationships table and we named it **treatment_histories**

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @momotsow, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did well to make the changes requested by the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**